src/pages/[platform]/reference/cli-commands/index.mdx

```diff
@@ -252,6 +252,7 @@ Optional parameters:
 - `--format`(_string_) (choices: `modelgen`, `graphql-codegen`, `introspection`) - Specifies the format of the GraphQL client code to be generated.
 - `--model-target` (_string_) (choices: `java`, `swift`, `javascript`, `typescript`, `dart`) - Specifies the modelgen export target. Only applies when the --format parameter is set to modelgen.
 - `--statement-target`(_string_) (choices: `javascript`, `graphql`, `flow`, `typescript`, `angular`) - Specifies the graphql-codegen statement export target. Only applies when the --format parameter is set to graphql-codegen.
+- `--statement-max-depth`(_integer_) - Specifies the maximum depth of the generated GraphQL statements. Only applies when the --format parameter is set to graphql-codegen.
```

Comment:
is this a valid CLI option? I'm not seeing it in the help, but if it is we should get it added to the help
```
ampx generate graphql-client-code

Generates graphql API code

Stack identifier
  --stack  A stack name that contains an Amplify backend                [string]

Project identifier
  --app-id  The Amplify App ID of the project                           [string]
  --branch  A git branch of the Amplify project                         [string]

Options:
  --debug             Print debug logs to the console [boolean] [default: false]
  --help              Show help                                        [boolean]
  --profile           An AWS profile name.                              [string]
  --out               A path to directory where config is written. If not provid
                      ed defaults to current process working directory. [string]
  --format            The format that the GraphQL client code should be generate
                      d in.
              [string] [choices: "modelgen", "graphql-codegen", "introspection"]
  --model-target      The modelgen export target. Only applies when the `--forma
                      t` parameter is set to `modelgen`
         [string] [choices: "java", "swift", "javascript", "typescript", "dart"]
  --statement-target  The graphql-codegen statement export target. Only applies
                      when the `--format` parameter is set to `graphql-codegen`
    [string] [choices: "javascript", "graphql", "flow", "typescript", "angular"]
  --type-target       The optional graphql-codegen type export target. Only appl
                      ies when the `--format` parameter is set to `graphql-codeg
                      en`
  [string] [choices: "json", "swift", "typescript", "flow", "scala", "flow-moder
                                                                  n", "angular"]
  --all               Show hidden options                              [boolean]

```

Reply:
It is and it should!

Reply:
would you mind filing a quick bug report for this in the backend repo? are there any other options we should add?

(in the future I'd like to generate this page from the CLI itself)